packages/server/src/ui/components/score-icon.jsx

```diff
@@ -6,11 +6,40 @@
 
 import {h} from 'preact';
 
-/** @param {{score: number}} props */
+
+/** @type {Record<LHCI.AssertCommand.Budget.TimingMetric, [number, number]>} */
+const SCORE_LEVEL_METRIC_THRESHOLDS = {
+  'first-contentful-paint': [2000, 4000],
+  'first-meaningful-paint': [2000, 4000],
+  'largest-contentful-paint': [2000, 4000],
+  'first-cpu-idle': [3000, 7500],
+  interactive: [3000, 7500],
+  'speed-index': [3000, 6000],
+  'total-blocking-time': [300, 600],
+  'cumulative-layout-shift': [0.1, 0.25],
+  'max-potential-fid': [100, 250],
+};
+
+/** @param {{score: number, diffs?: LHCI.AuditDiff[]}} props */
 export const ScoreIcon = props => {
-  const score = props.score;
-  if (score >= 0.9) return <i className="lh-score-pass" />;
-  if (score >= 0.5) return <i className="lh-score-average" />;
+  const { score, diffs } = props;
+  let result = score;
+  const numericDiff = diffs && diffs.find(diff => diff.type === 'numericValue');
+  if (numericDiff && numericDiff.type === 'numericValue') {
+    // @ts-ignore
+    const thresholds = SCORE_LEVEL_METRIC_THRESHOLDS[numericDiff.auditId];
+    if (thresholds) {
+      if (numericDiff.compareValue <= thresholds[0]) {
```

Comment:
why are you doing this? the score already has its own thresholds and we want them to match the actual score.

Reply:
With current implementation, CLS audit always get score of 1 for me and because it is always 1, it always show passing icon for CLS audit diff. I may be wrong with my observation for having score 1 for CLS. Please correct me if my observation is wrong here.

Reply:
@patrickhulce I just tried again with a page which has bad CLS score and I can see non 1 score for CLS, which means my assumption was wrong there. Let me fix the changes accordingly and come back.

Reply:
@patrickhulce I have reverted the changes for this file as I have verified that scores are working fine and we don't need to handle the thresholds separately for CLS.